_chapters/chapter_04.md

```diff
@@ -817,7 +835,7 @@ layout: rails_tutorial
       </li>
       ```
 
-      Rails gives us a `link_to` method we can use to create anchor elements.
+      Rails gives us a [`link_to`](http://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to) method we can use to create anchor elements.
```

Comment:
Can we replace this with a link to the official docs? http://api.rubyonrails.org/v5.0.0/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to